openedx/features/smart_referral/tests/test_views.py

```diff
@@ -0,0 +1,69 @@
+import mock
+from ddt import data, ddt
+from django.urls import reverse
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_403_FORBIDDEN
+from rest_framework.test import APITestCase
+
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+
+from .factories import SmartReferralFactory
+
+
+@ddt
+class SmartReferralInvitationAPIViewTest(APITestCase):
+
+    def setUp(self):
+        super(SmartReferralInvitationAPIViewTest, self).setUp()
+        self.user = UserFactory()
+        self.user.set_password('password')
+        self.user.save()
+        self.client.login(username=self.user.username, password='password')
+
+    @mock.patch('openedx.features.smart_referral.views.task_referral_and_toolkit_emails')
+    def test_send_initial_emails_success(self, mock_task_referral_and_toolkit_emails):
+        """Successfully send referral emails to two users and current user will receive a toolkit email"""
+
+        data = """[
```

Comment:
Why do we need it in a string form? can't we pass it directly? if not can we still define a list and then dump it to string? Actually I want to remove duplication of email, in this way we will define variables for email and use them at line `26` as well as `33`.

Reply:
also we already have a data type with test referral data `test_data_contacts.json`, can we some how use that data here, instead of recreating the data?